client/src/constants/permissions.js

```diff
@@ -0,0 +1,22 @@
+const SCOPES = {
```

Comment:
Minor: we might want to think about a good way to share constants that are used by both the frontend and backend. Out of scope for this ticket obviously, but it's not very DRY nor maintainable to have the same constants defined in both places.